modules/installing-ove-console.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-booting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-installing-ove-console_{context}"]
+= Configuring the cluster installation from the web console
+
+Once the machines have been booted from the ISO image and the rendezvous node has been selected, configure the rest of the cluster installation from the web console.
+
+.Prerequisites
+
+* You have the URL of the installation web console that was provided by the rendezvous node.
+
+.Procedure
+
+. In a web browser, go to the URL provided by the rendezvous node.
+
+. Configure your cluster in the *Cluster details* page:
```

Comment:
I didn't check, but maybe for this part you could reuse the existing Assisted UI docs?

Reply:
Unfortunately the assisted installer docs are on a separate repo on Gitlab, so direct reuse is not feasible in this case. Also, I noticed some slight deviation: for example the Assisted UI lets you check a box to optionally add a different pull secret, but in this workflow it seems like you always need to manually add one.